pallets/bitacross/src/custodial_wallet.rs

```diff
@@ -15,14 +15,15 @@
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 
 use codec::{Decode, Encode};
-use core_primitives::{Address20, Address33};
 use scale_info::TypeInfo;
 
+pub type PubKey = [u8; 33];
```

Comment:
Did you try it?

Are `ethereum_key.public_bytes` and `bitcoin_key.public_bytes` guaranteed to be convertible to this type?

Reply:
Yes, it's 
```
pub type CompressedPoint = GenericArray<u8, U33>;
```